docs/topics/native/native-cocoapods-libraries.md

```diff
@@ -285,4 +285,29 @@ kotlin {
         }
     }
 }
+```
+
+### Dependencies between Pods
+
+You can use the cinterop binding generated for another Pod when building a binding for the new Pod.
```

Comment:
I think the case for which it's required should be described here. Because we don't want users to use this DSL "just in case".

Reply:
Thank you! Please check the new intro